### PR TITLE
Update setup.py and add python 3.9 CI tests

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

**Comment by rwedge** [comment]
we could add this to the changelog

**Comment by gsheni** [comment]
@rwedge done.

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -14,11 +14,13 @@ Changelog
 **v2.0.0** Oct 13, 2021
     * Changes
         * Update primitives for compatibility with Featuretools 1.0.0 (:pr:`61`)
+        * Update setup.py with new and correct information (:pr:`72)
```
Comment (rwedge) [inline docs/source/changelog.rst:17]: this should be in the future release section

Comment (gsheni) [reply docs/source/changelog.rst]: My bad. Fixed

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -18,7 +20,7 @@ Changelog
         * Individual CI jobs will not cancel if other jobs fail (:pr:`67`)
 
     Thanks to the following people for contributing to this release:
-    :user:`rwedge`, :user:`thehomebrewnerd`
+     :user:`rwedge`, :user:`thehomebrewnerd`
```
Comment (rwedge) [inline docs/source/changelog.rst:23]: extra whitespace

Comment (gsheni) [reply docs/source/changelog.rst]: Fixed

**Review by rwedge — APPROVED** [review]
LGTM

**Pull request merged**